content/300-guides/200-deployment/110-deployment-guides/300-deploying-to-vercel.mdx

```diff
@@ -1,165 +1,320 @@
 
-![Vercel env var options](https://res.cloudinary.com/prismaio/image/upload/v1628761152/docs/jqcX0tV.png)
+   Vercel will now deploy your application. The `package.json` file in the root directory of the example project uses a [`vercel-build` script](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/node-js#custom-build-step-for-node.js) to run `prisma generate && prisma migrate deploy && next build`. This generates Prisma Client, applies the migrations, and then builds the Next.js project. This ensures that the generated Prisma Client in `node_modules/@prisma/client` is available to the Next.js serverless functions.
 
-Finally, click **Save**.
+### Result
 
-Note that this prevents Preview deployments from working, as they have no access to the database.
+When the build completes, you should see the following:
 
-In order to enable Preview deployment for pull requests, create a separate database for preview deployments, set the `DATABASE_URL` env var , and expose it **only** to Preview deployments.
+![Vercel deployment succeeded](images/300-50-deploy-to-vercel-success.png)
 
-> **Note**: because each pull request shares the same preview database, if you have multiple pull requests with diverging schema migrations, you will encounter problems applying them.
+You have successfully deployed the app to Vercel. Note that Vercel will build and deploy further commits to your GitHub repository.
 
-## <inlinecode>vercel/pkg</inlinecode> <inlinecode>ENOENT</inlinecode> error
+To check that the API returns data successfully, click on the image of the deployed application. This will take you to your application at `https://__PROJECT-NAME__.vercel.app/`, where `__PROJECT-NAME__` is the name that Vercel generates for the project.
 
-If you use [vercel/pkg](https://github.com/vercel/pkg) to package your Node.js project, then you might encounter an error like the following:
+When the page loads, click **Check API status**. If successful, you will see a result similar to the image below.
 
-```
-spawn /snapshot/enoent-problem/node_modules/.prisma/client/query-engine-debian-openssl-1.1.x ENOENT
+```code no-copy
+{"up":true}
 ```
 
-To avoid this error, add your Prisma query engine binary path to the `pkg/assets` section of your `package.json` file, as follows:
+## 2. Test your deployed application
 
-```json file=package.json copy
-{
-  "pkg": {
-    "assets": ["node_modules/.prisma/client/*.node"]
-  }
-}
-```
+You can now test your application. Open your application at `https://__PROJECT_NAME__.vercel.app` to interact with the API.
+
+### Steps
+
+1. Click **Load users with profiles**.
+
+   You do not yet have any data in the database, so the application returns an empty array:
+
+   ```code no-copy
+   []
+   ```
+
+   <Admonition type="info">
+
+   This API call uses Prisma to get all `users` in the database with their related `profiles`. The code is in the `pages/api/users.js` file in the project repository.
+
+   </Admonition>
```

Comment:
Nit: The admonition formatting here and further down is a bit weird as it makes these look like quotes.